mythril/laser/ethereum/instructions.py

```diff
@@ -1056,6 +1056,7 @@ def jumpi_(self, global_state):
                 states.append(new_state)
             else:
                 logging.debug("Pruned unreachable states.")
+        del global_state
```

Comment:
won't this be automatically be garbage collected?

Reply:
I think so but I am not totally sure, I added del everywhere as a desperate attempt to reduce memory and this got left off.